lib/actuators/include/DCMotor.h

```diff
@@ -1,14 +1,11 @@
 #pragma once
 
-#ifndef MOTOR_DEFAULT_FREQUENCY_HZ
-#define MOTOR_DEFAULT_FREQUENCY_HZ 244  // 10 kHz
-#endif
+#include "Actuator.h"
 
-#include "mbed.h"
-
-// Interface to control a standard DC motor with an H-bridge using a PwmOut and 2 DigitalOuts
+class DCMotor : public Actuator {
+ protected:
+  static constexpr int MOTOR_DEFAULT_FREQUENCY_HZ = 1000;  // TODO
 
-class Motor {
  public:
   typedef struct motorConfig {
```

Comment:
for config structs, please replicate what i did for the PID stuff. declare it in the namespace and just name it Config. It'll be addressed as DCMotor::Config so its obvious it related to dcmotor